src/jank/index.js

```diff
@@ -96,4 +104,16 @@ window.onload = () => {
       }
     );
   };
+
+  /**
+   * Redirects the user to the specified scorepad.
+   *
+   * @param {Object} scorepad the scorepad to redirect the user to
+   */
+  function redirectToScorepad(scorepad, sideName) {
```

Comment:
Theoretisch brauchen wir die Funktion hier gar nicht (die ist nur innerhalb von window.onload definiert, und kann von außerhalb gar nicht aufgerufen werden, also auch nicht in dieser Datei).

Unten können wir das dann einfach direkt zuweisen, ich schreibe da noch einen Kommentar.